docs/Monorepos-Overview.md

```diff
@@ -0,0 +1,97 @@
+
+- **Security concerns**: If your projects require different access controls or have varying levels of security requirements, a monorepo might not be suitable. Ensuring that sensitive parts of the codebase are protected can be more challenging in a monorepo setup.
+
+- **Large, distributed teams**: For large, distributed teams, a monorepo can become hard to manage. With many contributors, the repository can grow quickly, leading to longer build times and more complicated merges. In these cases, using separate repositories might be better, allowing different teams to work independently without slowing each other down.
+
+## Common monorepo tools
+
+| Tool                          | Pro                                                                                                                                       | Cons                                                                                                                             |
+| ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Yarn Workspaces [Recommended] | Built-in support for monorepos. <br>Simplifies dependency management across packages. <br>Excellent community support and documentation.  | Can be complex to configure initially. <br>Some issues with peer dependencies.                                                   |
+| Lerna                         | Great for managing multi-package repositories. <br>Supports independent versioning of packages. <br>Integrates well with Yarn Workspaces. | Slower for large codebases. <br>Requires additional configuration for CI/CD.                                                     |
+| Nx                            | Focuses on improving performance with advanced caching and task scheduling. <br>Great for large-scale enterprise applications.            | Steeper learning curve. <br>May be overkill for smaller projects.                                                                |
+| Turborepo                     | High performance with built-in caching and parallel task execution. <br>Simplifies monorepo management with minimal configuration.        | Less mature ecosystem compared to Yarn or Lerna. <br>Limited tooling support for some workflows.                                 |
+| Pnpm                          | Efficient disk space usage with unique package storage. <br>Fast installation times. <br>Built-in support for monorepos.                  | Less mainstream adoption, leading to potential issues with community support. <br>Some tooling might not fully support Pnpm yet. |
+
+## Common monorepo setups for software apps
+
+### Frontend and backend in a single monorepo
+
+Structure:
+
+- `/frontend`: Contains the React Native app and other frontend components.
```

Comment:
I think between 
 "## Common monorepo setups for software apps" and "### Frontend and backend in a single monorepo"  it would be good to have a short paragraph on how to choose your structure:

> The way that you organize your monorepo is entirely up to you. Often, teams will fall back to a "default" organization that looks like this:
>
> - `apps/my-app`
> - `packages/package-one`
> - `packages/package-two`
>
> But just because it's common, doesn't mean that its right.  You should think carefully and pick a structure that makes sense for your project and your team. Here are some general rules:
>
> 1. Aim to keep your structure as simple as possible -- if you can use a single `/packages` folder that's sometimes the easiest and best. Each package is entirely independent, so if you have a small number of packages, then you may not need a complicated structure.
> 2. Consider CI and tooling -- jobs are often easiest to target by directory, so if a group of packages needs special tooling, or will be released together, keep them in the same folder. 
>
> Below are some examples of structures that might work for you and your team.